packages/pds/src/db/message-queue/index.ts

```diff
@@ -51,6 +56,15 @@ export class SqlMessageQueue implements MessageQueue {
     this.cursorExists = true
   }
 
+  async ensureCaughtUp(): Promise<void> {
+    try {
+      await this.processAll()
+    } catch (err) {
+      log.error({ err }, 'error ensuring queue is up to date')
+    }
+    setTimeout(this.ensureCaughtUp, 60000) // 1 min
+  }
+
```

Comment:
Would be nice to have something like this so that we can teardown the message queue— otherwise processes running the queue wont naturally exit, and they'll need to be torn down with a bit of force.  We might have a couple similar cases like this dangling around, too.  (Pretty sure jest even notes them at the end of test runs.)
```suggestion
    this.ensureCaughtUpTimeout = setTimeout(this.ensureCaughtUp, 60000) // 1 min
  }

 destroy() {
   clearTimeout(this.ensureCaughtUpTimeout)
 }
```